subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -1,8 +1,17 @@
-# Copyright (c) 2018 Nordic Semiconductor
+# Copyright (c) 2025 Nordic Semiconductor
```

Comment:
I think you want to extend the copyright here instead of changing the original year